.github/workflows/main.yaml

```diff
@@ -128,7 +128,8 @@ jobs:
   docker-unittest-image:
```

Comment:
Codespell check fails because "neccessary ==> necessary" change in the file above, could you address this here ?

Reply:
done